packages/node/src/handlers.ts

```diff
@@ -410,7 +410,7 @@ export function requestHandler(
           .then(() => {
             _end.call(this, chunk, encoding, cb);
           })
-          .catch(e => {
+          .then(null, e => {
```

Comment:
Just out of curiosity: what's the advantage here of using `then(null, e)` instead of `catch`?

Reply:
`PromiseLike` type that we use doesnt have `catch` method. It's more generic variant of `Promise` implementation - https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_es5_d_.promiselike.html

Reply:
See https://github.com/getsentry/sentry-javascript/pull/2273 for more reference

Reply:
I'll check it out, thx 👍

Reply:
Also see my comment above: https://github.com/getsentry/sentry-javascript/pull/4667#discussion_r818199547